docs/docs/getting-started/bare-metal.md

```diff
@@ -26,6 +26,8 @@ Follow Canonical's instructions on [setting up Intel TDX on Ubuntu 24.04](https:
 </TabItem>
 </Tabs>
 
+Increase the `user.max_inotify_instances` sysctl limit by adding `user.max_inotify_instances=8192` to `/etc/sysctl.d/99-sysctl.conf` and running `sysctl -p`.
```

Comment:
According to my manpage, it should be `sysctl --system` to read from files in `/etc/sysctl.d/`, but I rarely use the tool.

Reply:
Yup, you're right, `-p` loads from `/etc/sysctl.conf` by default and this *won't* include `/etc/sysctl.d/99-sysctl.conf`. `--system` is better here.